app/views/orders/show.html.erb

```diff
@@ -39,17 +39,9 @@
 
   </table>
 
-  <div>
-    <div>
-      <p>Please leave special instruction below:</p>
-      <input type="text" name="instruction" placeholder="Instructions">
-      </div>
-
-      <div>
-      <p>Total: <%= @total %></p>
-      <%# button_to "Checkout", checkout_path, :class => "btn btn-default" %>
-      </div>
-    </div>  
+  <div class="instruction">
+    <p>Please leave special instruction below:</p>
+    <input type="text" name="instruction" placeholder="Instructions">  
```

Comment:
NB: ok but nemember that the instructions wouldn't be stored in the order table for now